modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/trust/trustTypes.scala

```diff
@@ -0,0 +1,16 @@
+package org.tessellation.sdk.infrastructure.trust
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Interval
+import fs2.data.csv.RowDecoder
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+
+object trustTypes {
+  case class TrustRating(value: Float)
+
+  object TrustRating {
+    implicit val rowDecoder: RowDecoder[TrustRating] = deriveRowDecoder
+  }
+
+  type TrustRatingRefined = Float Refined Interval.Closed[-1.0, 1.0]
```

Comment:
My next step is to incorporate the refined type. I may have to make a custom `RowDecoder` in order to do that.

Reply:
I think you can use map/contramap for that purpose. Anyway, again - it's not a correct order of doing things. Start from the domain part (model, types), then business logic, then talking with outside world (csv loader). That way, the CSV loader would work on the specific types that are final and refined.

Reply:
Implemented the refined type.